.npmignore

```diff
@@ -6,3 +6,4 @@ tmp/**/*
 .idea/**/*
 *.iml
 *.log
+coverage
```

Comment:
We have `files` in package.json already, just delete this file?

Reply:
npmignore is indeed abit errorprone, files in package.json ftw